src/06-longitudinal-and-multi-site-studies.md

```diff
@@ -71,11 +70,11 @@ sub-<label>/
 Optional: Yes
 
 In case of multiple sessions there is an option of adding additional
-participant key files describing variables changing between sessions. In such
-case one file per participant should be added. These files need to include
-compulsory `session_id` column and describe each session by one and only one
-row. Column names in per participant key files have to be different from group
-level participant key column names.
+`sessions.tsv` files describing variables changing between sessions. In such
+case one file per participant SHOULD be added. These files MUST include a
+`session_id` column and describe each session by one and only one row.
+Column names in `sessions.tsv` files MUST be different from group
+level participant key column names in the `participants.tsv` file.
```

Comment:
following from "participants.tsv":

> that MUST consist of `sub-<label>` identifying each row for a
participant, followed by a list of optional columns describing participants.

```suggestion
`session_id` column that MUST consist of `ses-<label>`, and that
MUST describe each session by one and only one row.
Column names in `sessions.tsv` files MUST be different from group
level participant key column names in the `participants.tsv` file.
```

Reply:
could you please also add a link back to "participants file" in the last two words of this sentence?

Reply:
Done